onnxruntime/core/framework/sequential_executor.cc

```diff
@@ -289,7 +289,7 @@ Status SequentialExecutor::Execute(const SessionState& session_state, const std:
       // call compute on the kernel
       VLOGS(logger, 1) << "Computing kernel: " << node_name_for_profiling;
 
-      kernel_begin_time = session_state.Profiler().StartTime();
+      kernel_begin_time = session_state.Profiler().Now();
```

Comment:
I know this is old code. But personally I feel this get kernel_begin_time should move down, and put it at the last step for the preparation, i.e., after CalculateTotoalInputSizes().

Reply:
Done.